week2/project/ecommerce/src/pages/ProductsPage/ProductsPage.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Categories from '../../components/product/categories'
+import Products from '../../components/product/products'
+
+
+
+export default function ProductsPage() {
+   const [category, setCategory] = React.useState('All items')
+
+   const handlerCategoryClick = (event) => {
+      setCategory(event);
+   }
+
+   return (
+      <div>
+         <h2 className='headTitle'>PRODUCTS</h2>
+         <Categories onClick={handlerCategoryClick} activeCategory={category} />
+         <Products prodCategory={category} />
```

Comment:
I think it's more organized to make folders for `categories` & `products` inside the `ProductsPage` folder because it will render inside ProductPage so it's easy to access it.

📝advise📝